logging/Limiting_Access_to_Logs.md

```diff
@@ -137,25 +136,25 @@ logging/bin/offboard.sh --namespace <i>namespace</i>  [--tenant <i>tenant</i>]
 </pre>
 
 - *namespace* is required. It specifies the Kubernetes namespace for which the 
-  corresponding Kibana-tenant space and access controls are removed.
+  corresponding OpenSearch Dashboards-tenant space and access controls are removed.
 - *tenant* specifies the SAS Viya tenant for which the corresponding 
-  Kibana-tenant space and access controls are removed.
+  OpenSearch Dashboards-tenant space and access controls are removed.
 
-As with the `onboard.sh` script, the Kibana-tenant space name is a 
-combination of the namespace name and the SAS Viya tenant name (if used). For example, if you ran the script and specified only `--namespace mynamespace`, the script would remove the Kibana-tenant space named `mynamespace`. If you ran the script and specified 
-`--namespace mynamespace --tenant mytenant1`, the script would remove the Kibana-tenant space named `mynamespace_mytenant1`. 
+As with the `onboard.sh` script, the OpenSearch Dashboards-tenant space name is a 
+combination of the namespace name and the SAS Viya tenant name (if used). For example, if you ran the script and specified only `--namespace mynamespace`, the script would remove the OpenSearch Dashboards-tenant space named `mynamespace`. If you ran the script and specified 
+`--namespace mynamespace --tenant mytenant1`, the script would remove the OpenSearch Dashboards-tenant space named `mynamespace_mytenant1`. 
 
 ## Managing User Accounts for Restricted Access
 
 ### Introduction
 
-You can create or delete Kibana accounts 
-that have access to only the log messages and Kibana-tenant space 
+You can create or delete OpenSearch Dashboards accounts 
```

Comment:
My bad.  The accounts are technically "OpenSearch accounts" and NOT "OpenSearch Dashboards accounts".  The tenant spaces are "OpenSearch Dashboard-tenant spaces"

Reply:
Corrections made.